common/src/main/java/com/redhat/cloud/notifications/openbridge/BridgeAuthService.java

```diff
@@ -14,12 +14,17 @@
  * Talk to the IDM server of openbridge to get a bearer token
  */
 @RegisterRestClient(configKey = "kc")
-@Path("/auth/realms/event-bridge-fm/protocol/openid-connect/token")
+@Path("/auth/realms/redhat-external/protocol/openid-connect/token")
 public interface BridgeAuthService {
 
     @POST
     @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
     @Produces(MediaType.APPLICATION_JSON)
-    Map<String, Object> getTokenStruct(String body,
-         @HeaderParam("Authorization") String authHeader);
+    Map<String, Object> getTokenStructWithClientPassword(String body,
+                                                         @HeaderParam("Authorization") String authHeader);
+
+    @POST
```

Comment:
Does this method and the other one target the same remote endpoint? I'm asking because there's no `@Path` on any of the methods here.

Reply:
It is the same endpoint. One of the two will go away in the future.